.mvn/maven.config

```diff
@@ -0,0 +1 @@
+-DaltSnapshotDeploymentRepository=repo.spring.io::default::https://repo.spring.io/libs-snapshot-local -P spring
```

Comment:
Not related to this PR, but wanted to mention it, I think we don't we need the -DaltSnapshotDeploymentRepository.  Thoughts?